README.md

```diff
@@ -24,6 +24,27 @@ action "Export SVG from Figma" {
 }
 ```
 
+**Example worklfow using yml**
```

Comment:
I think we should replace the old "example" and use this one instead. Can you make that update @schalkneethling ?

Reply:
Sure thing